geode-docs/developing/transactions/about_transactions.html.md.erb

```diff
@@ -22,61 +22,160 @@ limitations under the License.
 <a id="topic_jbt_2y4_wk"></a>
 
 
-This section covers the features of <%=vars.product_name%> transactions.
+This section introduces <%=vars.product_name%> transactions.
+Geode offers an API for client applications that do transactional work.
+While the implementation of the API does not provide
+applications a rigid adherence to all the ACID properties of transactions,
+it handles many situations.
+It is fast in comparison to the slow,
+locking methods of a traditional database.
+
+## Overview of the Application's Transaction
+
+There are two ways that an application can run a transaction:
```

Comment:
The developer is still using the transaction API in function, so I would not call them two different ways.